core-build/gulp-core-build/src/utilities/GCBTerminalProvider.ts

```diff
@@ -1,12 +1,15 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-import { GulpTask } from '@microsoft/gulp-core-build';
+import { GulpTask } from '../tasks/GulpTask';
```

Comment:
Can you separate the local import into a different block?

Reply:
Done.